src/iterative_sorting/iterative_sorting.py

```diff
@@ -22,7 +22,17 @@ def bubble_sort(arr):
         return bubble_sort(arr)
 
 
-# STRETCH: implement the Count Sort function below
 def count_sort(arr, maximum=-1):
```

Comment:
real nice solution; works even if array is not sequential.
believe it would still need an error handler for negative numbers